src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const array = [];
+  const newState = { ...state };
```

Comment:
Naming issue: The variable 'newState' should be named 'stateCopy' to better indicate that it is a copy of the state.